Source/EPUBCore/FREpubParser.swift

```diff
@@ -21,15 +21,22 @@ class FREpubParser: NSObject, SSZipArchiveDelegate {
     Returns a FRBook.
     */
     func readEpub(epubPath withEpubPath: String) -> FRBook {
+        
         epubPathToRemove = withEpubPath
         
         // Unzip   
         let bookName = (withEpubPath as NSString).lastPathComponent
-        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName)
-        SSZipArchive.unzipFileAtPath(withEpubPath, toDestination: bookBasePath, delegate: self)
+        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName+".dir")
```

Comment:
I guess because a directory named the same as the original epub (name.epub) seemed misleading. `.dir` symbolizes that its a directory and not the epub file. Thoughts?